test/generatortests.js

```diff
@@ -14,7 +15,7 @@ var runGeneratorTests = function(testcases) {
                 // It seems reasonable to expect that the generated xpath
                 // should parse back to the same object, although this may 
                 // not always hold true.
-                equal(parsed.toString(), xpath.parse(parsed.toXPath()).toString(), "" + i + " produced same result when reparsed.");
+                equal(parsed.toString(), xpath.parse(parsed.toHashtag()).toString(), "" + i + " produced same result when reparsed.");
```

Comment:
Changed since `toHashtag` should return the same xpath if we run into an xpath and can't translate it to a hashtag and it made adding hashtag tests easier